assignments/prototypes.js

```diff
@@ -102,7 +135,7 @@
   console.log(archer.greet()); // Lilith offers a greeting in Elvish.
   console.log(mage.takeDamage()); // Bruce took damage.
   console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
```

Comment:
Line 22 needs to be switched to `${this.name} was removed from the game. `. This will give you the log you are looking for.